### PR TITLE
TestOfBranch n.1

### PR DESCRIPTION
**Pull request by StefanoSidelRighi**
Following a Roberto request

### CODE REVIEW

**Comment by Menelao147** [comment]
The file is empty, could you add something?

File: TestOfBranch
Comment (StefanoSidelRighi) [inline TestOfBranch:1]: It's ok but it's not covering the lunch time

**Review by Menelao147 — CHANGES_REQUESTED** [review]
The file is empty, could you add something?

**Review by olivier-training-profile — APPROVED** [review]
Good job 😉

**Review by Menelao147 — APPROVED** [review]
Thanks for the changes!

**Comment by StefanoSidelRighi** [comment]
Release validated

**Pull request merged**